lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -904,12 +919,17 @@ class SkMaskFilterNamespace {
 }
 
 // This needs to be bound to top-level because SkPaint is initialized
-// with `new`. Also in Dart you can't write this:
+// with `new`.
+// Also in Dart you can't write this:
 //
-//     external SkPaint SkPaint();
+//     external SkPaint SkPaint.create();
 @JS('window.flutterCanvasKit.Paint')
+@staticInterop
 class SkPaint {
-  external SkPaint();
+  external factory SkPaint.create();
```

Comment:
Any reason we want a named factory? Named and unnamed factories get lowered to the same code anyways here.

Reply:
soooo much better, must smaller diff now. ptal.